src/ChainSafe.Gaming.Lootboxes.Chainlink/LootboxTypeInfo.cs

```diff
@@ -2,8 +2,14 @@ namespace ChainSafe.Gaming.Lootboxes.Chainlink
 {
     public class LootboxTypeInfo
     {
+        /// <summary>
+        /// Lootbox Type ID. (Common, Rare, Epic, etc.)
+        /// </summary>
         public uint TypeId { get; set; }
 
+        /// <summary>
+        /// Amount of the lootboxes.
```

Comment:
You sure it is amount of the lootboxes? Looks a bit odd tbh 🤔

Reply:
Ah that was the balance, updated the docs.